src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/jpa/PrisonerIepLevel.kt

```diff
@@ -34,4 +36,8 @@ data class PrisonerIepLevel(
   override fun isNew(): Boolean = new
 
   override fun getId(): Long = id
+
+  @Transient
+  @ReadOnlyProperty
+  val reviewDate: LocalDate = reviewTime.toLocalDate()
```

Comment:
This is a convenience property for when code only cares about the date not the datetime.

It's to avoid having to call `.reviewTime.toLocalDate()` all the times.